jest/private/jest_config_template.mjs

```diff
@@ -40,6 +42,10 @@ function _runfilesPath(shortPath) {
   );
 }
 
+function _execPath(execPath) {
```

Comment:
nit: confusing that the input is named the same as the return.

maybe name `_execrootRelative` ?

Reply:
right. It should be `resolveExecPath` since it is resolving to the absolute path given an execPath

Reply:
✅